### DISCUSSION TITLE
youtube-vinegar/

### DISCUSSION THREAD

**Discussion by giscus** [idarek/dariusz.wieckiewicz.org#109] [General]
# youtube-vinegar/

O tym, jak po kilku latach unikania ponownie wróciłem do oglądania YouTube, dzięki prostemu środkowi czyszczącemu, jakim jest ocet.

https://dariusz.wieckiewicz.org/youtube-vinegar/

**Comment by pleomati** [comment]
nie prościej zainstalowac ad blocker w przeglądarce.

**Reply by idarek** [reply]
> nie prościej zainstalowac ad blocker w przeglądarce.

Nie każdy AdBloker zablokuje reklamy na YouTube tak jak Vinegar. Podobnie jak Vinegar, przeglądarka DuckDuckGo zastępuje odtwarzacz YouTube własnym odpowiednikiem przez którego reklamy się nie przejdą.
